ui/helpers/utils/settings-search.js

```diff
@@ -64,12 +64,12 @@ function colorText(menuElement, regex) {
     let elemText = menuElement.innerHTML;
     elemText = elemText.replace('&amp;', '&');
     elemText = elemText.replace(
-      /(<span style="background:var(--brand-colors-yellow-yellow500)">|<\/span>)/gim,
+      /(<span style="background:#ffd33d">|<\/span>)/gim,
```

Comment:
looks like this would not be dark-mode compatible

Reply:
Agreed. Is there not a way to adjust the function so the regex doesn't count the variable OR use a class name instead of reverting back to the static hex value?
```suggestion
      /(<span class="settings-page__header__search__list__item__highlight">|<\/span>)/gim,
```

Then in [ui/pages/settings/index.scss](https://github.com/MetaMask/metamask-extension/blob/46b3f003a675516852886448195b2fe5fc4624da/ui/pages/settings/index.scss#L97) you could add 
```
&__highlight {
  background-color: var(--color-warning-muted);
}
```

Highlight would be more subtle but would be theme compatible 


https://github.com/MetaMask/metamask-extension/assets/8112138/fd0332ad-678e-4e25-abda-300ccc4bc935